style.module.scss

```diff
@@ -36,45 +91,32 @@
   }
 }
 
-.overviewInstallChoiceStep {
-  img {
-    max-width: 100px;
-    height: 60px;
-    margin: 0;
-    padding: 5px 0 5px 0;
-    border: none;
-    order: 1;
-  }
-
-  img, strong {
-    align-self: center;
-    text-align: center;
-  }
-}
-
 .overviewFeatureChoiceStep {
   flex-direction: column;
 }
 
-@media (min-width: 620px) {
-  .overviewInstallChoiceStep img {
-    order: 0;
-  }
+@media (min-width: 640px) {
```

Comment:
[note for reviewers] I'm changing the min-width of this. I'm not sure where 620px came from, but I think standard is more like 640px (`sm`). I'm picking this up from Tailwind: https://tailwindcss.com/docs/responsive-design

I also used 768px as min-width to become 3 buttons, which is `md` in Tailwind, as well as the starting size of tablets and small desktops.